docs/pages/kubernetes-access/helm/reference/teleport-kube-agent.mdx

```diff
@@ -388,6 +388,41 @@ This can be used for joining a Teleport agent to a Teleport cluster which does n
   One option might be to use Teleport's built-in [ACME support](./teleport-cluster.mdx#acme) or enable [cert-manager support](./teleport-cluster.mdx#highavailabilitycertmanager).
 </Admonition>
 
+## `tls`
+
+### `existingCASecretName`
+
+| Type | Default value |
+| - | - |
+| `string` | `""` |
+
+`tls.existingCASecretName` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
+This can be set to inject a root and/or intermediate CA so that Teleport can build a full trust chain on startup.
+
+You should create the secret in the same namespace as Teleport using a command like this:
```

Comment:
If these instructions apply in all cases, let's replace "should" with either "must" or something like "First, create a secret in the same namespace as your `teleport` pods using a command similar to the following:".

Otherwise, it's not clear why we're offering this suggestion using "should".

Reply:
Fixed